nextcloudappstore/core/views.py

```diff
@@ -19,13 +21,31 @@ class CategoryAppListView(ListView):
     model = App
     template_name = 'app/list.html'
 
-    def get_queryset(self):
+    def get_queryset(self, request):
```

Comment:
the request is available as self.request here. No need to redefine get and pass it to the method.